cmd/activator/main.go

```diff
@@ -297,3 +298,8 @@ func main() {
 	http1Srv.Shutdown(context.Background())
 	h2cSrv.Shutdown(context.Background())
 }
+
+func flush(logger *zap.SugaredLogger) {
```

Comment:
nit: the name and parameter makes me feel that this function only flush logs, but actually it also flush metrics.